controllers/releaseplanadmission/controller.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package releaseplanadmission
+
+import (
+	"context"
+
+	"github.com/redhat-appstudio/operator-toolkit/controller"
+	"sigs.k8s.io/controller-runtime/pkg/cluster"
+
+	"github.com/go-logr/logr"
+	"github.com/redhat-appstudio/release-service/api/v1alpha1"
+	"github.com/redhat-appstudio/release-service/cache"
+	"github.com/redhat-appstudio/release-service/controllers/utils/handlers"
+	"github.com/redhat-appstudio/release-service/controllers/utils/predicates"
+	"github.com/redhat-appstudio/release-service/loader"
+	"k8s.io/apimachinery/pkg/api/errors"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Controller reconciles a ReleasePlanAdmission object
+type Controller struct {
+	client client.Client
+	log    logr.Logger
+}
+
+//+kubebuilder:rbac:groups=appstudio.redhat.com,resources=releaseplansadmissions,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=appstudio.redhat.com,resources=releaseplanadmissions/status,verbs=get;update;patch
```

Comment:
Doesn't need RPs as well?

Reply:
True, not sure why it didn't fail before. Just `get;list;watch` on the resource, not resource/status, right?